src/main/java/org/springframework/hateoas/Link.java

```diff
@@ -138,6 +213,66 @@ public Link withSelfRel() {
 	}
 
 	/**
+	 * Returns a {@link Link} with the {@code hreflang} attribute filled out.
+	 *
+	 * @param hreflang
+	 * @return
+	 */
+	public Link withHreflang(String hreflang) {

```

Comment:
Lombok's `@Wither`? Should we have `withHref(…)` and `withRel(…)`, too, for consistency reasons?